src/components/submissions/formatters.ts

```diff
@@ -4,7 +4,7 @@ import { SubmissionRow } from "./submissions-table/SubmissionsTable";
 export const formatDate = (params: FormatterParams<unknown>) => params.value.format("D MMM YYYY");
 
 export const formatResultsCount = (resultsCount?: number) =>
-    `${resultsCount || 0} result${resultsCount === 1 ? "" : "s"}`;
+    `${!resultsCount ? `0 batch submissions` : `${resultsCount} result${resultsCount === 1 ? "" : "s"}`}`;
```

Comment:
I don't think the outer string is required
```suggestion
    resultsCount ? `${resultsCount} result${resultsCount === 1 ? "" : "s"}` : "0 batch submissions";
```